lua/nightfox/extra/wezterm.lua

```diff
@@ -4,9 +4,13 @@ local M = {}
 
 function M.generate(spec)
   local content = [[
-# Nightfox Wezterm Colors
-# Style: ${palette.meta.name}
-# Upstream: ${palette.meta.url}
+[metadata]
+name = "${palette.meta.name} Wezterm Colors"
```

Comment:
Change name to just palette name
```suggestion
name = "${palette.meta.name}"
```